app/Console/Commands/Installer.php

```diff
@@ -221,7 +220,7 @@ protected function updateEnvFile($key, $value)
         $envPath = app()->environmentFilePath();
         $payload = file_get_contents($envPath);
 
-        if ($existing = $this->existingEnv($key, $payload)) {
+        if ($existing == $this->existingEnv($key, $payload)) {
```

Comment:
A single ```=``` was intentional as it sets the ```$existing``` variable used in the next line if an existing value exists.